src/components/MovieSearch.js

```diff
@@ -26,6 +29,9 @@ function MovieSearch() {
     return isValid;
   };
 
+// this function will be called when user click on search icon or click on enter button.
```

Comment:
this function will be called when user click on search icon or hit enter button.